cdap-distributions/src/hdinsight/install.sh

```diff
@@ -119,6 +119,22 @@ updateFsAzurePageBlobDirForCDAP() {
     restartCdapDependentClusterServices
 }
 
+__waitForServiceState() {
+    local __svc=${1}
+    local __state=${2}
+
+    for i in {1..60} ; do
+      __currState=$(curl -s -u ${USERID}:${PASSWD} -i -H 'X-Requested-By: ambari' -X GET http://${ACTIVEAMBARIHOST}:${AMBARIPORT}/api/v1/clusters/${CLUSTERNAME}/services/${__svc} | grep \"state\" | awk '{ print $3}' | sed -e 's/"\(.*\)"[,]*/\1/')
+      if [ "${__state}" == "${__currState}" ]; then
```

Comment:
for a simple string comparison like this, ``[]`` is perfectly fine, and more portable.